README.md

```diff
@@ -309,6 +309,19 @@ clique:register_usage(["riak-admin", "handoff"], handoff_usage()),
 clique:register_usage(["riak-admin", "handoff", "limit"], fun handoff_limit_usage/0).
 ```
 
+### register_writer/2
+This is not something most applications will likely need to use, but the
+capability exists to create custom output writer modules. Currently you can
+specify the --format=[human|csv] flag on many commands to determine how the
+output will be written; registering a new writer "foo" allows you to use
+--format=foo to write the output using whatever corresponding writer module
+you've registered.
+
+Writing custom output writers is relatively undocumented right now, and the
+values passed to the write/1 callback may be subject to future changes. But,
+the clique_*_writer modules in the clique source tree provide good examples
```

Comment:
It turns out that GFM doesn't care (but the diff output does, because CODE SHARING AT GITHUB IS APPARENTLY HARD) but escaping these underscores and asterisk with a backslash would be a good idea.